dotnet/Services/ProductReviewService.cs

```diff
@@ -137,19 +137,16 @@ public async Task<AverageCount> GetAverageRatingByProductId(string productId)
             if (reviews != null)
             {
                 decimal totalRating = 0;
-
+                numberOfReviews = reviews.Count;
```

Comment:
Before we were not including reviews with a zero rating in the total.  Do we know that we will will always have a rating of at least one?